test/library.cpp

```diff
@@ -569,9 +569,12 @@ TEST_F(LibraryTest, MigrateBookmark)
     EXPECT_EQ(allBookmarks[4].getBookId(), bookId2);
     EXPECT_EQ(allBookmarks[5].getBookId(), bookId1+"_date_flavour");
 
-    ASSERT_EQ(lib->migrateBookmarks(bookId1, bookId2), 0); // No more bookId1 bookmark
+    // Suprisingly, `char *` make c++ prefers `migrateBookmarks(..., bool)` instead of `std::string`.
+    // [??] Is is a problem ? It should appear only if user use `char *` (so use C or literal string).
+    // A solution would be to rename one method, but which one ?
```

Comment:
Another solution is to use an `enum` instead of `bool` for the parameter type, which will also convert the somewhat cryptic call `migrateBookmarks(..., true)` to a more self-explaining `migrateBookmarks(..., ALLOW_DOWNGRADE)`

Reply:
Indeed. This is a far better solution.